Enterprise-CMCS/eAPD
# Nz/4304 Update Budget and FFP for MMIS

Resolves #4304 

### Description
MMIS APDs do not need the Estimated Quarterly Expenditure Table in the Budget and FFP page.

### Significant changes or possible side effects
Figma link of how the Budget and FFP page should look after removing the EQE table:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=601%3A12578

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A HITECH APD | On the Budget and FFP page | The EQE Table should be visible | cypress |
| A MMIS APD | On the Budget and FFP page | The EQE Table should **not** be visible | cypress |

### Steps to manually verify this change

1. Go to `Activity > Budget and FFP` in HITECH APD and confirm table is there.
2. Go to `Activity > Budget and FFP` in MMIS APD and confirm table is **NOT** there.

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
